package.json

```
@@ -33,7 +33,8 @@
   },
   "dependencies": {
     "@actions/core": "^1.2.6",
-    "@actions/github": "^4.0.0"
+    "@actions/github": "^4.0.0",
+    "simple-git": "^2.21.0"
```

I think we can use [this method](https://octokit.github.io/rest.js/v18#repos-create-or-update-file-contents) in the currently added dependencies
```suggestion
    "@actions/github": "^4.0.0"
```

---

Yeah, will remove